src/core/importType.js

```diff
@@ -8,10 +8,6 @@ function constant(value) {
   return () => value
 }
 
-function isAbsolute(name) {
```

Comment:
instead of removing this, could you export it and then import it in `no-absolute-path`?